ui/app/styles/components/exec.scss

```diff
@@ -0,0 +1,146 @@
+.tree-and-terminal {
```

Comment:
Not a blocker, but I've been trying to keep component sass files limited to one root class that matches the file name. 

I wonder if there is a sensible way to refactor this file into its components without just creating file bloat and introducing patterns that are never going to be used elsewhere.

Reply:
ah good to know, thanks. I opened [this issue](https://github.com/hashicorp/nomad/issues/7477) to come back for this.